### ISSUE TITLE
"G, go to specific slide" can handle a logical page number, not a physical?

### ISSUE THREAD

**Issue by simplygood** [dannyedel/dspdfviewer#179]
Since the physical pdf page numbers of a beamer pdf with overlays do not match their logical page numbers. It would be great if I can go to a logical page number with 'G' shortcut. It is just quite impossible to guess a physical page when I have many overlays in pdf.

**Comment by dannyedel** [comment]
Makes sense : )

 I will see that I implement this (shoudn't be much difficulty, since poppler offers a [get page by string lookup][1] function)

[1]: https://people.freedesktop.org/~aacid/docs/qt5/classPoppler_1_1Document.html#a6393be3d1009747509f17b528bbadecd

**Comment by cristobaltapia** [comment]
Hi, 
I would also think that this would be a nice enhancement. Are you still planning to implement it?

**Comment by dannyedel** [comment]
@cristobaltapia Yes I do want to implement it, unfortunately my day job currently keeps me incredibly busy, so I cannot give any kind of ETA.

If you want to take over and implement it, I'd be happy to accept a pullrequest.
